BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,5 +1,18 @@
 {% extends 'BraincraftedBootstrapBundle:Form:bootstrap.html.twig' %}
 
+{% block form %}
```

Comment:
I didn't want to use a flashbag in this case because it would have implied a modification of the php code. By using this method, I ensure that every time a form has an error, there will be an error message.

@itkg-ngilain I will check the `form_errors` block